### PR TITLE
Update getting started guide

### PR DESCRIPTION
**Pull request by shamsher31**
**What this PR does / why we need it**:
Few nits for getting started guide.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Additional information**

**Special notes for your reviewer**

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Getting Started
 
-This is a guide on how to get started with Cluster API Provider BringYourOwnHost. To learn more about cluster API in more
+This is a guide on how to get started with Cluster API Provider BringYourOwnHost(`BYOH`). To learn more about cluster API in more
```
Comment (dharmjit) [inline docs/getting_started.md:3]: Nit: Inline code highlighting introduces extra spaces.
```suggestion
This is a guide on how to get started with Cluster API Provider BringYourOwnHost(BYOH). To learn more about cluster API in more
```

**Review by dharmjit — APPROVED** [review]
LGTM!

**Review by anusha94 — APPROVED** [review]
lgtm

**Pull request merged**